unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/TextureResource/AssetPromise_TextureResource.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using UnityEngine;
+
+namespace DCL
+{
+    public class AssetPromise_TextureResource : AssetPromise<Asset_TextureResource>
```

Comment:
Now I understand, I thought of it as a rename, but it was a different promise altogether.

The `AssetPromise_Texture` code already reuses textures with the same settings, I'm pretty sure that this promise can be removed and everything will be reused correctly.

The would not include the base64 code as its a deprecated feature and should not be part of SDK 7.